firmware/Reload Pro.cydsn/ui.c

```diff
@@ -108,13 +112,24 @@ const menudata choose_readout_menu = {
 	}
 };
 
+const menudata tools_menu_data = {

```

Comment:
You shouldn't actually have to write a function for the tools menu - just use the generic menu function like the main menu does. Then you define a state_func for the battery ISR entry only.